sympy/vector/tests/test_vector.py

```diff
@@ -3,9 +3,9 @@
      sin, cos, Function, Integral, Derivative, diff, integrate
 from sympy.vector.vector import Vector, BaseVector, VectorAdd, \
      VectorMul, VectorZero
-from sympy.vector.coordsysrect import CoordSysCartesian
+from sympy.vector.coordsysrect import CoordSys3D
 
-C = CoordSysCartesian('C')
+C = CoordSys3D('C')
```

Comment:
Tests for deprecated functions should be left in place until the deprecated function is removed. This ensures that the deprecated function doesn't break on any future changes to SymPy that occur as long as the deprecated function is in place.